website/templates/emails/contributor_added.txt.mako

```diff
@@ -1,6 +1,6 @@
 Hello ${user.fullname},
 
-You have been added as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
+${node.contributors[0]} has added you as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
```

Comment:
I believe `node.contributors[0]` is the user who created the project, but any administrator on the project can add contributors. 

Can you pass in the name of the user who added the contributor to this email? I don't think the auth is being passed to `notify_added_contributor` so you'll need to do that in order to pass `auth.user.fullname` to the email template.